command.go

```diff
@@ -217,16 +216,16 @@ func resInt(x interface{}) (int, error) {
 }
 
 func buildCommand(fields []string) []byte {
-	var b bytes.Buffer
-	b.WriteByte('*')
-	b.WriteString(strconv.Itoa(len(fields)))
-	b.WriteString("\r\n")
+	b := make([]byte, 0, 64)
```

Comment:
I guess it's not worth it to estimate the size more precisely based on the fields.